test/mocha/10-api.js

```diff
@@ -1,6 +1,203 @@
 /*!
  * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
  */
-describe('api', () => {
-  it('should work');
+import {api, delegateRootZcap} from './helpers.js';
+import {
+  presentations,
+  testExchanges
+} from '../mock.data.js';
+import {
+  shouldBeDelegatedZcap,
+  shouldError,
+  shouldHaveInteractService,
+  shouldHaveVpForStep,
+  shouldHaveVpWithTransactionId,
+  shouldNotError
+} from './assertions.js';
+
+const exchange = '/exchanges/:exchangeId';
+const exchangeInstance = '/exchange-instances/:exchangeInstanceId';
+
+describe('API', () => {
+  describe(exchange, () => {
+    it('should get a VP from a valid service with only one step', async () => {
+      const path = 'exchanges/oneStep';
+      const {response, error, data} = await api.post({path});
+      shouldNotError({response, error, data, path});
+      const {
+        verifiablePresentationRequest
+      } = testExchanges.oneStep.steps.initial;
+      shouldHaveVpForStep({data, path, verifiablePresentationRequest});
+    });
+    it('should error if service doesn\'t exist.', async () => {
+      const path = 'exchanges/nonExistent';
+      const {response, error, data} = await api.post({path});
+      //FIXME this is a client error where the wrong service was
+      // requested and should probably return 404.
+      const expected = {status: 500};
+      shouldError({response, error, data, path, expected});
+    });
+    it('should error if service doesn\'t have an initial step.', async () => {
+      const path = 'exchanges/noInitialStep';
+      const {response, error, data} = await api.post({path});
+      const expected = {status: 500};
+      shouldError({response, error, data, path, expected});
+    });
+    it('should error if in interact a service has an invalid type.',
```

Comment:
These test descriptions all still have periods at the end -- could you please remove to keep with consistent styling?

Reply:
addressed here: https://github.com/digitalbazaar/bedrock-vc-exchanger/pull/10/commits/ddca903f15d3e9961a2b50a89ca4d3fd7ac6e1e8